channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/logging/JaicpConversationLogger.kt

```diff
@@ -0,0 +1,74 @@
+package com.justai.jaicf.channel.jaicp.logging
+
+
+import com.justai.jaicf.api.BotRequest
+import com.justai.jaicf.channel.jaicp.DEFAULT_PROXY_URL
+import com.justai.jaicf.channel.jaicp.JSON
+import com.justai.jaicf.channel.jaicp.dto.JaicpBotRequest
+import com.justai.jaicf.channel.jaicp.dto.JaicpLogModel
+import com.justai.jaicf.channel.jaicp.dto.jaicpNative
+import com.justai.jaicf.channel.jaicp.http.ChatAdapterConnector
+import com.justai.jaicf.channel.jaicp.http.HttpClientFactory
+import com.justai.jaicf.channel.jaicp.JaicpPollingConnector
+import com.justai.jaicf.channel.jaicp.JaicpWebhookConnector
+import com.justai.jaicf.logging.LoggingContext
+import com.justai.jaicf.helpers.logging.WithLogger
+import com.justai.jaicf.logging.ConversationLogObfuscator
+import com.justai.jaicf.logging.ConversationLogger
+import io.ktor.client.*
+import io.ktor.client.features.logging.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+/**
+ * Implements [ConversationLogger] for JAICP Application Console.
+ *
+ * Any channel connected via [JaicpWebhookConnector] or [JaicpPollingConnector] will produce logs to JAICP.
+ * If [JaicpConversationLogger] is added to array of loggers, but [BotRequest] is not received from JAICP, nothing will happen.
+ *
+ * @see [JaicpPollingConnector]
+ * @see [JaicpWebhookConnector]
+ * @see [JaicpBotRequest]
+ * */
+class JaicpConversationLogger(
+    accessToken: String,
+    logObfuscators: List<ConversationLogObfuscator> = emptyList(),
```

Comment:
Not sure. We have only one conversationLogger-related property - logObfuscators. And we have required property accessToken, all the other properties are optional and created just in case anyone needs it.

It looks a bit dirty, but I see no point creating other data classes with 1-2 values.